ios/FluentUI/Command Bar/CommandBar.swift

```diff
@@ -0,0 +1,287 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+@objc(MSFCommandBar)
+open class CommandBar: UIView {
+    // Hierarchy:
+    //
+    // closeButton
+    // containerView
+    // |--layer.mask -> containerMaskLayer (fill containerView)
+    // |--subviews
+    // |  |--scrollView (fill containerView)
+    // |  |  |--subviews
+    // |  |  |  |--stackView
+    // |  |  |  |  |--buttons (fill scrollView content)
+
+    // MARK: Public Properties
+
+    @objc public weak var delegate: CommandBarDelegate?
+
+    // MARK: Private Properties
```

Comment:
Nit: Follow the ordering described at aka.ms/swift-guide 
https://microsoft.github.io/swift-guide/FileContentOrdering.html